packages/rn-tester/js/examples/TextInput/TextInputExample.ios.js

```diff
@@ -422,6 +422,7 @@ function OnPaste(): React.Node {
           appendLog(JSON.stringify(e.nativeEvent.dataTransfer.types));
           setImageUri(e.nativeEvent.dataTransfer.files[0].uri);
         }}
+        pastedTypes={['fileUrl', 'image', 'string']}
         placeholder="MULTI LINE with onPaste() for PNG and TIFF images"
```

Comment:
It seems this placeholder text is incorrect now, since it accepts more than just images?